python/CreatePipWhl.cmake

```diff
@@ -53,6 +53,30 @@ if (PYTHON)
         # 1. Get the NGRAPH_VERSION exported from nGraph CMake
         # 2. If #1 isn't possible, then determine this by looking at the 
         # dependency list?
+        execute_process(COMMAND 
```

Comment:
While you are at it - would you mind parameterize the version numbers i.e., `.0.4.0` --> NGRAPH_REL_VERSION or something. For now we will hardcode this but in future - we will figure out a way to get this from nGraph programmatically. Thanks :)

Reply:
Actually nevermind. I am doing a PR to update the nGraph version - I will do this as part of that.